### ISSUE TITLE
Hibernation

### ISSUE THREAD

**Issue by TKGNET** [tsebring/ArkSavegameToolkitNet#2]
Hi Tobias,
ty for the huge amount of work you put into translating qowyns java code to C#.

Have you planned to integrate hibernation entries any time ? It is not used on dedicated servers, but is default for single player and non-dedicated servers (few players).

Pete

**Comment by tsebring** [comment]
Hello!

What are hibernation entries? Are you able to elaborate on the problem you experience with extracting singleplayer saves?

If you have a savegame I can look at that would help in tracking down the problem.

If you want you can contact me on Discord, my tag is Tobias#5051.

**Comment by tsebring** [comment]
Support for reading hibernation entries is already implemented in ark-savegame-toolkit:
https://github.com/Qowyn/ark-savegame-toolkit/blob/master/src/main/java/qowyn/ark/ArkSavegame.java
https://github.com/Qowyn/ark-savegame-toolkit/blob/master/src/main/java/qowyn/ark/HibernationEntry.java

**Comment by TKGNET** [comment]
That's what I told you on sunday on Discord, it's your decision to implement it or not.

**Comment by tsebring** [comment]
Yes. This is the issue tracking system. All information relevant to solving an issue gets added here :)
